http-api/Cargo.toml

```diff
@@ -31,6 +31,10 @@ async-trait = "0.1"
 ethers-core = "0.6.3"
 fastrand = "1.7.0"
 chrono = { version = "0.4.19", features = ["serde"] }
+axum = { version = "0.5.3", features = [ "headers" ] }
+axum-server = { version = "0.3", features = ["tls-rustls"] }
+hyper = { version ="0.14.17", features = ["server"] }
+tower-http = { version = "0.3.0", features = ["trace", "cors"] }
```

Comment:
Did we check if these work with `default-features = false` as well? maybe saves a bit of build time.

Reply:
Hm... I didn't, but it's pretty fast tbh.